.redhat-ci.sh

```diff
@@ -7,13 +7,18 @@ mount --make-rshared /
 
 if [ -f /run/ostree-booted ]; then
     if [ ! -e /var/tmp/ostree-unlock-ovl.* ]; then
-        ostree admin unlock
+	# Until overlayfs and selinux get along, use remount
```

Comment:
Can we keep doing the unlock for Fedora?  Should be basically `if grep -q ID=fedora /etc/os-release` ?